src/controller/CHIPDeviceController.cpp

```diff
@@ -3041,7 +3041,7 @@ void DeviceCommissioner::PerformCommissioningStep(DeviceProxy * proxy, Commissio
 
         CHIP_ERROR err = CHIP_NO_ERROR;
         ChipLogProgress(Controller, "SendCommand kWiFiNetworkEnable, supportsConcurrentConnection=%d",
-                        params.GetSupportsConcurrentConnection().Value());
+                        params.GetSupportsConcurrentConnection().ValueOr(false));
```

Comment:
At this point, shouldn't `params.GetSupportsConcurrentConnection()` have a value based on the earlier read?  Or is that only done by autocommissioner and other commissioners are not ensuring the right (new) invariants are followed?

Reply:
@bzbarsky-apple 
Yes, you are correct. Of course, crashes do not occur in general sequences. (This occurred during testing using the previous code.) However, if the code has changed and this value can be empty, I uploaded a PR because I thought it would be necessary to prevent a crash due to this code that simply shows the log.

Reply:
In that case, it would be better to log separately for the "missing value" case, which should probably be logged as error.

Reply:
It was already added by andy advise.